ot/da.py

```diff
@@ -1123,8 +1267,9 @@ def __init__(self, reg_e=1., mode="unsupervised", max_iter=1000,
         self.log = log
         self.mapping = mapping
         self.metric = metric
-        self.distribution = distribution
+        self.distribution_estimation = distribution_estimation
         self.method = "sinkhorn"
+        self.out_of_sample_map = out_of_sample_map
 
     def fit(self, Xs=None, ys=None, Xt=None, yt=None):
         """Build a coupling matrix from source and target sets of samples
```

Comment:
I see that in every fit and transform, all of the parameters are initialized to None. However, I think that at least Xs (and Xt for the inverse transform) should be required.